Tools/Frame_params/iflight-chimera7.param

```diff
@@ -0,0 +1,77 @@
+#NOTE: iFlight Chimera7 params for ArduPilot Copter 4.2
+
+ACRO_RP_EXPO,0.6
+ACRO_RP_RATE,600
+ACRO_Y_EXPO,0.6
+ACRO_Y_RATE,600
+ATC_ACCEL_P_MAX,281062.5
+ATC_ACCEL_R_MAX,301679.8
+ATC_ACCEL_Y_MAX,72000
+ATC_ANG_PIT_P,18
+ATC_ANG_RLL_P,18
+ATC_ANG_YAW_P,11
+ATC_RAT_PIT_D,0.001879093
+ATC_RAT_PIT_FLTD,60
+ATC_RAT_PIT_FLTT,30
+ATC_RAT_PIT_I,0.09663188
+ATC_RAT_PIT_P,0.09663188
+ATC_RAT_RLL_D,0.001453624
+ATC_RAT_RLL_FLTD,75
+ATC_RAT_RLL_FLTT,30
+ATC_RAT_RLL_I,0.07706726
+ATC_RAT_RLL_P,0.07706726
+ATC_RAT_YAW_D,0.02
+ATC_RAT_YAW_FF,0.0043
+ATC_RAT_YAW_FLTD,60
+ATC_RAT_YAW_FLTE,1.887445
+ATC_RAT_YAW_FLTT,30
+ATC_RAT_YAW_I,0.01927038
+ATC_RAT_YAW_P,0.1927038
+BATT_AMP_OFFSET,-0.04308
+BATT_AMP_PERVLT,52.706
+BATT_CURR_PIN,11
+BATT_LOW_VOLT,21.6
+BATT_MONITOR,4
+COMPASS_AUTO_ROT,0
+COMPASS_CUS_PIT,-15

```

Comment:
add 4.2 in the file name, and maybe create a non-4.2 version?

Reply:
ok fixed